providers/providers.go

```diff
@@ -202,7 +205,7 @@ func (pm *ProviderManager) run(proc goprocess.Process) {
 		gcQueryRes <-chan dsq.Result
 		gcSkip     map[string]struct{}
 		gcTime     time.Time
-		gcTimer    = time.NewTimer(pm.cleanupInterval)
+		gcTimer    = timers.NewGaussian(pm.cleanupInterval, cleanupInervalDeviation)
```

Comment:
Brief explanation of what this does?

Reply:
Creates a timer that will fire on average every 1 hour with 10 minutes of deviation (+ or -).